### PR TITLE
Update overview.md to use the right link for face recognition

### PR DESCRIPTION
**Pull request by hustcalm**
Use the right link for face registration

### CODE REVIEW

**Comment by Jak-MS** [comment]
@laujan 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by laujan — APPROVED** [review]

File: articles/ai-services/content-understanding/video/overview.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ Content extraction for video includes transcription, shot detection, key frame e
 * **Shot detection**: Identifies segments of the video aligned with shot boundaries where possible, allowing for precise editing and repackaging of content with breaks exactly on shot boundaries.
 * **Key frame extraction**: Extracts key frames from videos to represent each shot completely, ensuring each shot has enough key frames to enable Field Extraction to work effectively.
 * **Face grouping**: Grouped faces appearing in a video to extract one representative face image for each person and provides segments where each one is present. The grouped face data is available as metadata and can be used to generate customized metadata fields.
-* This feature is limited access and involves face identification and grouping; customers need to register for access at [Face Registration](https://aka.ms/faceregistration).
+* This feature is limited access and involves face identification and grouping; customers need to register for access at [Face Registration](https://aka.ms/facerecognition).
```
Comment (Jak-MS) [inline articles/ai-services/content-understanding/video/overview.md]:
```suggestion
* This feature is limited access and involves face identification and grouping; customers need to register for access at [Face Recognition](https://aka.ms/facerecognition).
```
@laujan - should the link text be updated as well?  the destination page is Face Recognition.

Comment (laujan) [reply articles/ai-services/content-understanding/video/overview.md]: Yes, @Jak-MS. Good catch!

**Review by laujan — APPROVED** [review]

**Comment by laujan** [comment]
#sign-off

**Pull request merged**